### ISSUE TITLE
Empty POST request parameters while sending request with content-type: application/json header

### ISSUE THREAD

**Issue by tooleks** [huang-yi/laravel-swoole-http#1]
The application receiving empty `POST` parameters while sending `HTTP` `POST` request with
`content-type: application/json` header.

**Steps To Reproduce**

Create a route on fresh Laravel installation:

```php
Route::post('/', function () {
    return request()->all();
});
```
Run swoole server, then send `HTTP` `POST` request via `cURL`:

```bash
curl -X POST \
  'http://localhost:1215/' \
  -H 'accept: application/json' \
  -H 'content-type: application/json' \
  -d '{"data": "test"}'
```

Output:

```bash
[]
```

**Software**
* Ubuntu 17.04
* PHP 7.1
* Swoole 2
* Laravel 5.5

Anyway, thanks for a great package!

**Comment by huang-yi** [comment]
Thank you for your feedback.

I did some tests in different Swoole versions (including v2.0.1, v2.0.9), but I didn't see the situation you described, and it works fine.

![swoole-1](https://user-images.githubusercontent.com/6905173/31701345-67eb648a-b402-11e7-92bb-5f08eabf796a.png)
![swoole2](https://user-images.githubusercontent.com/6905173/31701347-6a30e436-b402-11e7-927b-16dbf44ee9ae.png)
![swoole3](https://user-images.githubusercontent.com/6905173/31701371-879e84a6-b402-11e7-93ce-54a27b5afe1f.png)


**Can you provide an exact version number of the Swoole? I will do more tests.**

**Comment by tooleks** [comment]
Oh, it looks like I'm using an older version of swoole. Sorry, for bad info, that I've provided.

```bash
php -r "echo phpversion('swoole');"
```
```bash
1.9.21
```
I will try to upgrade to the newest one and re-run test.

**Comment by tooleks** [comment]
Well, I have updated swoole from `1.9.21` to `2.0.10` and that solved my issue. That was my fault, sorry about that.

But I found another strange behavior.

When I try to access the `POST` parameter via `\Illuminate\Http\Request::get` method under the swoole I recieved `null`, while under the php-fpm I received an actual `POST` parameter value.

```php
Route::post('/', function () {
    return request()->get('data'); // null under swoole
});
```

```php
Route::post('/', function () {
    return request()->get('data'); // (string) "test" under php-fpm
});
```

I found a workaround by replacing `\Illuminate\Http\Request::get` to `\Illuminate\Http\Request::input`, but it requires the codebase changes and breaks BC beetween php-fpm and swoole.
